stream-chat-android-state/.env

```diff
@@ -0,0 +1,3 @@
+STREAM_LOG_LEVEL=ALL
```

Comment:
Why do we need this file? 🤔

Reply:
Probably not, not sure where and when this was introduced, will remove it.